### ISSUE TITLE
Email notification

### ISSUE THREAD

**Issue by ppastur** [soulteary/docker-cronicle#23]
Hi I have just installed cronicle and initial testing seems excellent.  Thank you for your work.  I am however not receiving emails even though I configured an email address for the user.  I suspect I need to set up email settings such as SMTP server, port, user, password etc.  Can you please help me work out how to do that?
Thanks

**Comment by HSUHAOBM** [comment]
I have tested Gmail successfully.
1. Access the container.
2. Edit the configuration file at /opt/cronicle/conf/config.json.
3. Restart the container.

**Comment by ppastur** [comment]
Yes I managed to do that as well but sadly, if the container gets deleted, the config will need to be redone and I can seem to bind the volume containing the config.json so I can back it up and re-attach with the new container.

**Comment by deuts** [comment]
Just copy the config.json to the host, then bind it back to the container as you run it.

**Comment by ppastur** [comment]
///How would I do that using docker-compose?

**Comment by Rumpick** [comment]
@ppastur You probably already have something like "~/cronicle/plugins:/opt/cronicle/plugins" in your volumes, right? add one for "~/cronicle/config.json:/opt/cronicle/conf/config.json" to the compose file.
